### PR TITLE
Update docs with conda install instruction

### LINKED ISSUE #8: Add `textgrad` to conda-forge channel
A conda install option for textgrad could be helpful. It not only helps with installation of textgrad in conda environments, but also facilitates easier adoption of textgrad among downstream libraries that depend on textgrad and that may want to add themselves to conda-forge channel in future.

I have already started a conda-forge PR to add `textgrad`. Once the PR is merged, you can install textgrad as follows:

```sh
conda install -c conda-forge textgrad
```

Please assign this issue to me. I will update the documentation, once the upstream conda-forge PR is merged.

- PR: https://github.com/conda-forge/staged-recipes/pull/26673

### PR DESCRIPTION
**Pull request by sugatoray**
- [x] added conda-forge badges
- [x] updated badges
- [x] added conda install instruction
- [x] add conda-forge feedstock info
- [x] added pip + repo (for bleeding edge) install instruction

Closes #8

### CODE REVIEW

**Comment by sugatoray** [comment]
cc: @vinid @mertyg

File: README.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+[![PyPI](https://img.shields.io/pypi/v/textgrad)][#pypi-package]
+[![Conda - Platform](https://img.shields.io/conda/pn/conda-forge/textgrad?logo=anaconda&style=flat)][#conda-forge-package]
+[![Conda (channel only)](https://img.shields.io/conda/vn/conda-forge/textgrad?logo=anaconda&style=flat&color=orange)][#conda-forge-package]
+
+[#license-gh-package]: https://lbesson.mit-license.org/
+[#arxiv-paper-package]: https://arxiv.org/abs/2406.07496
+[#docs-package]: https://textgrad.readthedocs.io/en/latest/?badge=latest
+[#pypi-package]: https://pypi.org/project/textgrad/
+[#conda-forge-package]: https://anaconda.org/conda-forge/textgrad
+<!--- BADGES: END --->
```
Comment (sugatoray) [inline README.md:18]: Rendered Preview:

image

File: README.md [hunk 2]
```diff
@@ -110,12 +119,30 @@ You can run them directly in Google Colab by clicking on the links below.
...
+
+> :bulb: The conda-forge package for `textgrad` is maintained [here](https://github.com/conda-forge/textgrad-feedstock).
+
+**Bleeding edge installation with `pip`**:
+
+```sh
+pip install git+https://github.com/zou-group/textgrad.git
+```
+
+See [here](https://pip.pypa.io/en/stable/cli/pip_install/) for more details on various methods of pip installation.
```
Comment (sugatoray) [inline README.md:144]: Rendered Preview: [See README.md on working branch](https://github.com/sugatoray/textgrad/blob/update_docs/README.md)

image

**Comment by vinid** [comment]
looks awesome! thank you!

**Review by vinid — APPROVED** [review]

**Pull request merged**